### ISSUE TITLE
Found a cool website for figuring out which VPN location to use for your datacenter

### ISSUE THREAD

**Issue by Bananaman** [Soreepeong/XivAlexander#153]
https://is.xivup.com/adv

I only checked the IPs for 1 datacenter. I took all the IPs and put them into https://www.maxmind.com/en/geoip-demo/ and these are the result:

- Light Datacenter (EU): All the Light servers are in:
> "Düsseldorf,
> North Rhine-Westphalia,
> Germany,
> Europe"

So if you find out the location, and use a good (low ping) VPN in that datacenter location, and also use the XIVAlexander plugin, you should have the best ping possible in this game.

**Comment by Soreepeong** [comment]
*Assuming that WHOIS records at least have accurate internet service providers.*

But do note that if you are not in the same country (state, if you're at a huge country like United States), physical distance matter less; just go with the VPN nodes closest to well-known hosting providers, like Amazon Web Services, Microsoft Azure, or Google Cloud.

### Japanese Datacenters at Tokyo hosted by [Square Enix](https://whois.domaintools.com/124.150.157.158)
* neolobby01.ffxiv.com
* neolobby03.ffxiv.com
* neolobby05.ffxiv.com

### North American Datacenters at Sacremento hosted by [NTT Communications](https://www.ntt.com/en/services/data-center/nexcenter/us.html)
* neolobby02.ffxiv.com
* neolobby04.ffxiv.com
* neolobby08.ffxiv.com

### European Datacenters at Frankfurt hosted by [KDDI Germany](https://www.de.kddi.com/products_services/detail2/telehouse.html)
* neolobby06.ffxiv.com
* neolobby07.ffxiv.com

### Chinese Datacenters at Beijing hosted by [Tencent Cloud](https://whois.domaintools.com/109.244.5.5)
* ffxivlobby01.ff14.sdo.com
* ffxivlobby05.ff14.sdo.com
* ffxivlobby07.ff14.sdo.com

### Korean Datacenter at Bundang hosted by [KT IDC](https://whois.domaintools.com/183.111.189.3)
* lobbyf-live.ff14.co.kr

**Comment by Bananaman** [comment]
@Soreepeong That's very good info, that's the list of each datacenter login server.

The website I have given above lists the actual server IPs which may be better for ping tests (actual game servers, instead of the login server).

They seem to use 1 lobby/login server per datacenter (such as Light EU), and then 1 server per actual Realm (such as Phoenix on Light EU).

The interesting thing is that I tried one of your lobby servers, 195.82.50.9, and did a GEOIP lookup of that too. It came up yet again as Duesseldorf, not Frankfurt.

**Comment by Soreepeong** [comment]
Yeah, it's probably a blanket name for their service region by KDDI Germany. Also, I am 100% sure that they got lobby servers and contents servers in the very same network, so it shouldn't matter at all!

**Comment by Bananaman** [comment]
@Soreepeong Yeah network is the same but I was thinking about server load. Login server is probably always optimal ping. But eh, maybe there is no difference. :) 

By the way. I tried multiple other GeoIP websites. All say Duesseldorf.

- The most famous lookup site: https://www.maxmind.com/en/geoip-demo/ 
- http://geoiplookup.net/ip/195.82.50.9
- https://geoip.com

I stopped there because every site says the IP is in Duesseldorf.

Usually those sites are correct, because they base it on how internet subnets are divided. How do we know that anything is still in Frankfurt? There were many years old FF15 news about that but after that news I just heard players repeating it?

If anyone can confirm that it is still in Frankfurt with official proof, we can conclude if GeoIP is right or not. For my personal IP, GeoIP knows my exact region of my city to a huge accuracy. But maybe the same isn't true for corporate networks.

**Comment by Soreepeong** [comment]
It's more of that it doesn't matter. For pretty much everyone outside Germany (which majority of xiv player is) they would care whether server is in Germany or not since Germany is only a size of one North American state IMO. 

By the way, check this out: https://www.de.kddi.com/files/topics/500_ext_01_0.pdf

KDDI got their ICT & Systems Integration team at Dusseldorf, and Datacenters at Frankfurt. It's probably just a blanket IP address registration on RIPE.

**Comment by Bananaman** [comment]
@Soreepeong Oh looking at KDDI's website was a really smart idea. Here is a map of all their datacenter locations:

https://www.de.kddi.com/products_services/detail1/global_datacenter.html

Turns out KDDI is a Japanese provider with global locations, so I bet Square uses them for every datacenter.

**Comment by Soreepeong** [comment]
Nope, they go with NTT America for American datacenters! NTT and KDDI are huge in Japan, so it makes sense.

**Comment by Bananaman** [comment]
@Soreepeong Yeah sorry I just noticed that your post had listed the service providers. Great work on your list! :) I have edited my posts to remove confusion.
